graphrag/index/flows/create_final_embeddings.py

```diff
@@ -0,0 +1,194 @@
+                column_to_embed="name",
+                filename="create_final_entities_name_embeddings",
+                callbacks=callbacks,
+                cache=cache,
+                storage=storage,
+                base_text_embed=base_text_embed,
+                message="Entities embeddings (name)",
+            )
+
+        # Embed name+description in entities table
+        if entity_description_embedding in embedded_fields:
+            await run_embeddings(
+                data=entities_embeddings,
+                column_to_embed="name_description",
+                filename="create_final_entities_description_embeddings",
+                callbacks=callbacks,
+                cache=cache,
+                storage=storage,
+                base_text_embed=base_text_embed,
+                message="Entities embeddings (description)",
+            )
+
+        # Embed title in community reports table
+        if community_title_embedding in embedded_fields:
+            await run_embeddings(
+                data=community_reports_embeddings,
+                column_to_embed="title",
+                filename="create_final_community_reports_title_embeddings",
+                callbacks=callbacks,
+                cache=cache,
+                storage=storage,
+                base_text_embed=base_text_embed,
+                message="Community reports embeddings (title)",
+            )
+
+        # Embed summary in community reports table
+        if community_summary_embedding in embedded_fields:
+            await run_embeddings(
+                data=community_reports_embeddings,
+                column_to_embed="summary",
+                filename="create_final_community_reports_summary_embeddings",
+                callbacks=callbacks,
+                cache=cache,
+                storage=storage,
+                base_text_embed=base_text_embed,
+                message="Community reports embeddings (summary)",
+            )
+
+        # Embed full_content in community reports table
+        if community_full_content_embedding in embedded_fields:
+            await run_embeddings(
+                data=community_reports_embeddings,
+                column_to_embed="full_content",
+                filename="create_final_community_reports_full_content_embeddings",
+                callbacks=callbacks,
+                cache=cache,
+                storage=storage,
+                base_text_embed=base_text_embed,
+                message="Community reports embeddings (full_content)",
+            )
+
+    return pd.DataFrame()
+
+
+async def run_embeddings(
+    data: pd.DataFrame,
+    column_to_embed: str,
+    filename: str,
+    callbacks: VerbCallbacks,
+    cache: PipelineCache,
+    storage: PipelineStorage,
+    base_text_embed: dict | None = None,
+    message: str | None = None,
+) -> None:
+    """All the steps to generate single embedding."""
+    if base_text_embed:
+        log.info("%s", message)
```

Comment:
Oh, check for msg. What will happen if it is null? Would still print something or just ignore it?

Reply:
I'm not really a fan of having this message arg anyway. It only exists for info logging, which is a bit weird to put in an "api". How about just logging using the filename, which contains all of the relevant info?